src/components/Article/Article.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const Article = ({ title, text, date }) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <>
+      <h2 data-cy="title">{title}</h2>
```

Comment:
You deleted className here className="title" and the styles don't apply to your project, I saw that while checking your demo (https://prnt.sc/k0_foZ4Ws9Sn)

Reply:
agree